redaxo/src/addons/be_style/plugins/customizer/pages/system.customizer.php

```diff
@@ -67,7 +67,7 @@
 
 // load config
 
-/** @var array{codemirror_theme: string, codemirror_darktheme: string, codemirror-selectors: string, codemirror-options: string, codemirror: int, codemirror-langs: int, codemirror-tools: int, labelcolor: string, showlink: int, codemirror-autoresize?: bool} $config */
+/** @var array{codemirror_theme?: string, codemirror_darktheme?: string, codemirror-selectors?: string, codemirror-options?: string, codemirror: int, codemirror-langs?: int, codemirror-tools?: int, labelcolor: string, showlink: int, codemirror-autoresize?: bool} $config */
```

Comment:
so richtig optional sind sie eigentlich nicht. dadurch, dass sie in der default-config in der package.yml definiert sind, sollten die immer existieren.

Reply:
ach ok, sehe die `isset`-Aufrufe, die dann eigentlich überflüssig wären. hmm

Reply:
aber an anderen stellen in der datei wird wiederum davon ausgegangen, dass sie gesetzt sind. was eigentlich wie gesagt ok ist.

Reply:
apropos

<img width="1215" alt="grafik" src="https://user-images.githubusercontent.com/120441/179391286-3855c52f-9818-41d8-87fa-e5d567540315.png">